pkg/kubelego/kubelego.go

```diff
@@ -340,5 +349,46 @@ func (kl *KubeLego) paramsLego() error {
 	} else {
 		kl.legoWatchNamespace = watchNamespace
 	}
+
+	keyType := os.Getenv("LEGO_KEY_TYPE")
+	switch keyType {
+	case kubelego.KeyTypeRsa:
+		kl.legoKeyType = kubelego.KeyTypeRsa
+		break
+	case kubelego.KeyTypeEcc:
+		kl.legoKeyType = kubelego.KeyTypeEcc
+		break
+	default:
+		kl.legoKeyType = kubelego.KeyTypeEcc
```

Comment:
I'd prefer not to change the default here, as I'm unsure how this would play with existing certificates.